app/src/main/res/layout/fragment_favorite.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why do we wrap everything in a framelayout? 
A constraint layout should do it most of the time as parent